src/promptflow-tools/promptflow/tools/yamls/openai_gpt4v.yaml

```diff
@@ -0,0 +1,49 @@
+promptflow.tools.openai_gpt4v.chat:
+  name: OpenAI GPT-4V
+  description: Use OpenAI GPT-4V to leverage vision ability.
+  type: custom_llm
+  module: promptflow.tools.openai_gpt4v
+  function: chat
+  default_prompt: |
+    # system:
+    As an AI assistant, your task involves interpreting images and responding to questions about the image.
+    Remember to provide accurate answers based on the information present in the image.
+
+    # user:
+    Can you tell me what the image depicts?
+    ![image]({{image_input}})
+  inputs:
+    connection:
+      type:
+        - OpenAIConnection
+    model:
+      type:
+        - string
```

Comment:
It should be a enum?

Reply:
I am not sure. If it is an enum, we may have to update the enum definition each time OpenAI release a new model.

Reply:
we can use enum and `allow_manual_entry` contract to solve the issue.